### DISCUSSION TITLE
Please add Swipe Sensitivity "Very High"

### DISCUSSION THREAD

**Discussion by jabberwocky7** [rxhanson/Multitouch-Community#140] [Ideas]
Thanks for the great work. 
I am enjoying Multitouch so far. 
It works just as good as BTT and I plan on switching completely to Multitouch.
However, I would like to have a "Very High" option for the swipe sensitivity because even with "High" I have to swipe quite a bit to trigger the action.
Please consider.

**Comment by rxhanson** [comment]
Thanks for the feedback.

Multitouch swipes are little different from BTT and different from macOS.

Try removing your touches sooner. The implementation is this way because the swipes are treated as instantaneous, rather than continuous; you're triggering a singular action so it's just a quick swipe and remove touches.

**Reply by jabberwocky7** [reply]
Thank you for the tip. I took your advice and now's it's working perfectly! Please ignore this request.
